scripts/buyV1Covers.js

```diff
@@ -1,3 +1,4 @@
+require('dotenv').config();
```

Comment:
This is redundant because `require('hardhat');` already does this (this require is done in hardhat config).

Reply:
It was necessary for it to work with the local deployment. Without it, it crashes saying it cannot find the contract account/address. 🤔